### PR TITLE
Cyborg non direct "combat" rebalance. Airlock seal buffed, borg RCD nerfed.

### PR DESCRIPTION
**Pull request by GuillaumePrata**
## About The Pull Request
Lately, we had a few PRs and a lot of conversation around Borg combat, how it is binary and really dated compared to what we have for humans right now.
That is a giant can of worms that I don't want to touch, but there is another side of dealing with Rogue Silicons that is often ignored because it simply isn't viable enough to bother IMO and that is "environmental preparation" if you can call it that way...

To give a simple comparison, when Xenos are announced everyone goes around welding vents and airlocks to maint, it's my favorite part of dealing with Xenos as crew and they also suffer from a binary combat like Silicons do.

## Pneumatic Airlock Seal Buff
You know the thing? I bet you don't... it's this little fella here:
<details>
<summary>Pneumatic seal</summary>

</details>
If they are applied to an airlock, it will lock it until the seal is removed, and it can only be removed by humanoids.

You can print them on the Eng and Sec lathes, they are first tier tech and are expansive but IMO, the main reason that they are never used is that they are Bulky, if you want to seriously lock down an area against Silicons/Xenos/Simple mobs you either need to drag a crate with these or do multiple trips carrying by hand.

I changed them to be medium sized so you can reasonably put some in your bag and added them to the Science lathe, as Science should be IMO, the first dept to deal with rogue silicons/xeno mob spam.

## Engborg RCD nerf
Right now the Engborg is the best silicon to subvert by a far, unless you have some niche assassination plan that a Medborg might do better, not like the Engborg is bad at this anyway since they have the stun arm...

The issue outside of combat is that Engborgs can RCD down reinforced walls in 5 seconds and their roundstart cell allows them to do this 5 times which is more than enough to break in even on the AI sat, the area with multiple layers of Rwalls.

This also makes the Pneumatic seals stupidly useless, why would you lock down the Vault/Tcomms/Armory/whatever other sabotage the Silicons are doing if they can just RCD down the Rwalls faster than the protolathe can print 1 seal...

So now Engborgs can't deconstruct Rwalls with their RCD.

## Why It's Good For The Game
Dealing with Silicons is really dated and right now boils to nuking them with the console/Ions/flash as you can't really stop them otherwise, we have a few tools and tricks to help like the seal or cutting AI + access wires on airlocks but no one bothers because the RCD exists on its current state.

On the other hand, Silicons thrive on non direct combat, sabotaging Tcomms, unlinking the Ore Silo to stop the crew's supply, cutting down power, etc, etc.

So with these, the crew has a way to deal with Silicons by limiting their mobility when they are working alone without making them weaker if they have a Human or a Malf AI helping them.

## Changelog

:cl: Guillaume Prata
balance: Pneumatic airlocks seals are normal sized now and can be printed at the Science lathe.
balance: After dealing with another weekly subverted Engborg spacing the armory. CentCom's Robotics division decided to downgrade the Engborg RCD by removing it's Reinforced Wall deconstruction.
/:cl:

### CODE REVIEW

**Comment by Gamer025** [comment]
Hmm maybe make the RCD being able to deconstruct Rwalls a late game upgrade?
Like put it into some quite far away research node and make it cost quite some materials.
That way borgs can't mess around with stuff at roundstart and if someone decides to give them the upgrade its on them for allowing the borg to deconstruct Rwalls.

**Comment by Ghommie** [comment]
> 
> Hmm maybe make the (cyborg) RCD being able to deconstruct Rwalls a late game upgrade?

It ought to be either locked behind emagging and malf AIs or straightway removed. Engiborgs already have the tools to deconstruct rwalls "just as the founding fathers intended".

**Comment by Coastfront** [comment]
Leave syndie sabotage borgs able to rcd down r-walls so they still atleast have some amount of purpose other than "i exist to shock doors and be slightly irritating"

**Comment by necromanceranne** [comment]
> > Hmm maybe make the (cyborg) RCD being able to deconstruct Rwalls a late game upgrade?
> 
> It ought to be either locked behind emagging and malf AIs at most or straightway removed. Engiborgs already have the tools to deconstruct rwalls "just as the founding fathers intended".

if it's an emag/malf ability, that's mostly undermining this change, doesn't it?

**Comment by Ghommie** [comment]
> 
> if it's an emag/malf ability, that's mostly undermining this change, doesn't it?

I suppose you are right but I'm pretty ambivalent on it.

**Comment by ATH1909** [comment]
> So now Engborgs can't deconstruct Rwalls with their RCD.

>:(

**Comment by MrDoomBringer** [comment]
can engiborgs RCD away doors with pneumatic seals?

**Comment by GuillaumePrata** [comment]
> can engiborgs RCD away doors with pneumatic seals?

Nope, they will need a human to remove it, to break through a nearby wall/window, ask their Malf AI to explode the airlock or whatever other creative solutions they come up with.

**Comment by JohnFulpWillard** [comment]
> 
> 
> Hmm maybe make the RCD being able to deconstruct Rwalls a late game upgrade? Like put it into some quite far away research node and make it cost quite some materials. That way borgs can't mess around with stuff at roundstart and if someone decides to give them the upgrade its on them for allowing the borg to deconstruct Rwalls.

There's no reason for engieborgs to be able to RCD reinforced walls away if normal RCD's never have that ability.

**Comment by Serxule** [comment]
> There's no reason for engieborgs to be able to RCD reinforced walls away if normal RCD's never have that ability.

as far as i know the reason normal rcds cant is because it would mean greytiders can (and did) just spend 10 seconds to get into caps office, armory, etc, whereas borgs are stopped from doing that by laws

**Comment by Mothblocks** [comment]
Wish these were two separate PRs

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -598,7 +598,7 @@
 			. += "There's a [note.name] pinned to the front..."
 			. += note.examine(user)
 	if(seal)
-		. += "It's been braced with a pneumatic seal."
+		. += "It's been braced with a pneumatic airlock seal."
```
Comment (Mothblocks) [inline code/game/machinery/doors/airlock.dm:601]:
```suggestion
		. += "It's been braced with \a [seal]."
```

`seal` is an item. Double-check this one for me.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by GuillaumePrata** [comment]
> Wish these were two separate PRs

For the sake of atomicity or do you have issues with either side?
I just don't think either change would have an impact on how we deal with silicons without the other.

File: code/game/machinery/doors/airlock.dm (hunk 1 above)
Comment (GuillaumePrata) [reply code/game/machinery/doors/airlock.dm]: It works and now we can have upgraded seals (please don't)

**Comment by Mothblocks** [comment]
Atomicity, I don't have objections to either of these

**Pull request merged**